### PR TITLE
Add Web PubSub live tests

### PR DESCRIPTION
**Pull request by conniey**
# Description

Adding support for live tests in Azure Web PubSub.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/webpubsub/tests.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        - name: azure-messaging-webpubsub-client
+          groupId: com.azure
+          safeName: azuremessagingwebpubsubclient
+      TimeoutInMinutes: 60
+      SupportedClouds: 'Public,UsGov,China'
+      EnvVars:
+        AZURE_LOG_LEVEL: 2
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```
Comment (benbp) [inline sdk/webpubsub/tests.yml:20]: Please change this to the config below, which reduces our dependency on secrets and also supports sovereign cloud testing should that ever be required:

```suggestion
        AZURE_CLIENT_ID: $(WEBPUBSUB_CLIENT_ID)
        AZURE_CLIENT_SECRET: $(WEBPUBSUB_CLIENT_SECRET)
        AZURE_TENANT_ID: $(WEBPUBSUB_TENANT_ID)
        AZURE_SUBSCRIPTION_ID: $(WEBPUBSUB_SUBSCRIPTION_ID)
```

Comment (alzimmermsft) [reply sdk/webpubsub/tests.yml:20]: @benbp should we be updating this everywhere in out tests as I believe all deployments create these AAD credential variables?

**Review by alzimmermsft — APPROVED** [review]
Approving to allow this to be merged but the change Ben is recommended should be done first.

File: sdk/webpubsub/tests.yml (hunk 1 above)
Comment (benbp) [reply sdk/webpubsub/tests.yml:20]: @alzimmermsft yes `$(aad-azure-sdk-test-client-id)` is a deprecated secret. What you can also do is update your test base classes that look up these variables to look for either AZURE_* or <service prefix>_*, the latter which you can get from the `AZURE_SERVICE_DIRECTORY` environment variable.

**Review by benbp — APPROVED** [review]

**Pull request merged**